samples/opencl/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+include(FindOpenCL.cmake)
```

Comment:
opencv buildbot on linux64 have cmake less then 3.1 what cause fatal error at cmake configuration step (in sample CMakeLists.txt cmake_minimum_required(VERSION 3.1) command is used).